tests/test_distributions.py

```diff
@@ -370,4 +358,4 @@ def test_int_uniform_distribution_asdict():
 def test_int_log_uniform_distribution_asdict():
     # type: () -> None
 
-    assert EXAMPLE_DISTRIBUTIONS["ilu"]._asdict() == {"low": 2, "high": 12, "step": 2}
+    assert EXAMPLE_DISTRIBUTIONS["ilu"]._asdict() == {"low": 2, "high": 12}
```

Comment:
Could you add a test case to ensure that `IntLogUniform` raises `FutureWarning` when it receives the `step` argument?

The following is just an example:
```python

def test_int_log_uniform_distribution_deprecation() -> None:
    with pytest.warns(FutureWarning):
        distributions.IntLogUniformDistribution(low=1, high=100, step=2)
```

Reply:
Thank you for suggesting codes! I have added the test.